src/ng/compile.js

```diff
@@ -2956,6 +2929,12 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
           (tag !== "img" && (attrNormalizedName === "src" ||
                             attrNormalizedName === "ngSrc"))) {
         return $sce.RESOURCE_URL;
+      } else if (tag === "img" && (attrNormalizedName === "src" ||
```

Comment:
If #14019 is landed, `audio`, `video` and `source` should also be added to the list, right?